OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -458,6 +458,39 @@ public void TestActivateExperimentNotRunning()
 
             Assert.IsNull(variationkey);
         }
+
+        [Test]
+        public void TestActivateWithTypedAttributes()
```

Comment:
This test case is not verifying the payload generated from EventBuilder is returned correctly or not. 
Create an expectedEventBuilder class and call EventBuilder, and compare it.

Reply:
Added payload comparison test case for typed attributes in EventBuilderTest.cs.